skbio/stats/power.py

```diff
@@ -565,104 +612,6 @@ def confidence_bound(vec, alpha=0.05, df=None, axis=None):
     return bound
 
 
-def bootstrap_power_curve(test, samples, sample_counts, ratio=None,
```

Comment:
Can you please add this function back and deprecate it? At a minimum we need to give users warning a full release in advance before something is removed. See `skbio.util.flatten` for an example of a deprecated function.